ui/pages/settings/index.scss

```diff
@@ -193,19 +193,8 @@
 
     @media screen and (max-width: $break-small) {
       display: block;
-      background-image: url('/images/caret-left-black.svg');
-      width: 18px;
-      height: 18px;
-      opacity: 0.5;
-      background-size: contain;
-      background-repeat: no-repeat;
-      background-position: center;
       margin-right: 16px;
       cursor: pointer;
-
-      [dir='rtl'] & {
```

Comment:
I think we still want this, or something like it, so that the caret appears correctly in languages that are read right to left

Reply:
oh 🤔  interesting. I didn't see how the direction was added here. Looking into this

Reply:
I see now. Thanks for the catch! 
<img width="362" alt="Screenshot 2022-03-22 at 12 58 12 AM" src="https://user-images.githubusercontent.com/20778143/159405231-924ff103-2e88-4180-8e95-74ded50d2dd7.png">

Reply:
This is just for `[dir='rtl'] & {` right? The rest of the removed css is okay to remove!

Reply:
yep! Thanks for confirming @georgewrmarshall! 

There's 1 more thing that's a little off after these changes. The caret no longer lines up with the content. These caret buttons have a larger target area which is great for usability. Is it okay to leave it as is?

Reply:
I think this is ok to leave like this!